### PR TITLE
fgsl 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by grhawk**
**Porting from homebrew/science in order to add a new Formula(fgsl@1) in the following pr**

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fgsl.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  url "https://www.lrz.de/services/software/mathematik/gsl/fortran/download/fgsl-1.2.0.tar.gz"
+  sha256 "00fd467af2bb778e8d15ac8c27ddc7b9024bb8fa2f950a868d9d24b6086e5ca7"
+
+  depends_on "pkg-config" => :build
+  depends_on "gcc"
+  depends_on :fortran
+  depends_on "gsl"
+
+  def install
+    ENV.deparallelize
```
Comment (ilovezfs) [inline Formula/fgsl.rb]: This needs to be reported and fixed upstream before we can merge the formula.

File: Formula/fgsl.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class Fgsl < Formula
+  desc "Fortran bindings for the GNU Scientific Library"
+  homepage "https://www.lrz.de/services/software/mathematik/gsl/fortran/"
+  url "https://www.lrz.de/services/software/mathematik/gsl/fortran/download/fgsl-1.2.0.tar.gz"
+  sha256 "00fd467af2bb778e8d15ac8c27ddc7b9024bb8fa2f950a868d9d24b6086e5ca7"
+
+  depends_on "pkg-config" => :build
+  depends_on "gcc"
```
Comment (ilovezfs) [inline Formula/fgsl.rb]: This needs to build with Xcode's clang to be in homebrew/core.

File: Formula/fgsl.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  def install
+    ENV.deparallelize
+
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "CC=gcc-7",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "check"
```
Comment (ilovezfs) [inline Formula/fgsl.rb]: This line should be removed as we only run `make check` and `make test` for security software.

File: Formula/fgsl.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  depends_on "gsl"
+
+  def install
+    ENV.deparallelize
+
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "CC=gcc-7",
+                          "--prefix=#{prefix}"
+    system "make"
```
Comment (ilovezfs) [inline Formula/fgsl.rb]: This is probably not needed since we won't be running `make check`.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/fgsl.rb (hunk 1 above)
Comment (grhawk) [reply Formula/fgsl.rb]: Ok... I understand... but most of the fortran sofware are not able to be compiled in parallel.

**Comment by ilovezfs** [comment]
Actually given this only has 28 downloads in the last 90 days in `homebrew/science`, it doesn't qualify for migration (we set the minimum at 50 https://github.com/Homebrew/homebrew-science/issues/6331#issuecomment-331691989). please consider hosting it in [your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap).

Sorry to reject your first contribution to Homebrew Core. We look forward to your next pull request.

**Comment by grhawk** [comment]
No problem... I understand... ;)

**Pull request closed without merging**